songs/form.directive.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+(function(){
+  angular
+  .module("songs")
+  .directive("songsForm", function() {
+    return {
+      templateUrl: "songs/_song.form.html",
+      replace: true,
+      restrict: "C",
+      scope: {
+        scope: "=",
+        formType: "@"
+      },
```

Comment:
what part of the directive is giving you an issue? do you get an error? I'm seeing an extra comma in here which might be a cause..